procedures/auxtel/reduction_procedures/Parabolic_Focus_Analysis.ipynb

```diff
@@ -0,0 +1,585 @@
+{
```

Comment:
<p>Maybe just add a line underneath that says if only dealing with imaging data then skip to the imaging section.</p><p>Also, in the imaging section below, add that it should be skipped if dealing with spectra.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
👍